tests/requirements.txt

```diff
@@ -3,3 +3,7 @@ coverage
 pyflakes>=2.1.1
 flake8
 isort
+python-dateutil
+mock
+cachetools
+https://github.com/divio/aldryn-forms/archive/master.zip
```

Comment:
Why is this not:     "aldryn-forms",	as removed before? if there's ever a need to do anything out of the ordinary we should add a comment to allow anyone to reverse later, exactly like the lxml comments

Reply:
I've added a comment explaining why.